helm_deploy/hmpps-resettlement-passport-api/files/stubs/resettlement-passport-and-delius-api/__files/getPrisonerByNomsNumber-G9981UK-200-response.json

```diff
@@ -1,3 +1,3 @@
 {
-    "crn": "X596670"
+    "crn": "X705049"
 }
```

Comment:
Missing eof newline 😛

Reply:
Not sure any of our json files have them...

Reply:
Putting in a new ticket to fix all of them so they are consistent